Defaults timeout values in the Settings

rahulKQL opened the pull request:
Defaults timeout values in the Settings

rahulKQL committed:
Documented defaults timeout values in the Bigtable Settings

Updated timeouts in `EnhnaceDBigtableStubSettings` as `BigtableClientSettings` settings methods are deprecated

romlogic commented on google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java:
```diff
@@ -43,12 +43,25 @@
  * Settings class to configure an instance of {@link BigtableDataClient}.
  *
- * <p>Sane defaults are provided for most settings:
+ * <p>Some defaults are provided for most settings:
```
Since you have made this change here, could you change line 242 too to "some defaults".  " Initializes a new Builder with **sane** defaults for all settings." 

hey, as discussed, there is "sane default" and you might want to change it back. :-)

rahulKQL committed:
javadoc fix

rahulKQL committed:
Addressing feedback comments

- Listed Description for straming and non-streaming operation.
- Tried explaning RPC vs Total timeout
- Tried explaining retryDelayMultiplier.
- Added default values on each setting operation's javadoc.
- Added bulkMutateRowsSettings description.

rahulKQL committed:
Updated Javadoc according to feedback

rahulKQL committed:
addressing feedback comments
